editor/gui/editor_file_dialog.h

```diff
@@ -52,8 +52,29 @@ class EditorFileDialog : public FileDialog {
 
 public:
 #ifndef DISABLE_DEPRECATED
+	// These enums need to exist with their original names, to be registered with backwards-compatible hashes for GDExtension.
+	enum Access {};
+	enum DisplayMode {};
+	enum FileMode {};
```

Comment:
Does making these aliases via `using` work? Something like:

```suggestion
	using Access = FileDialog::Access;
	using DisplayMode = FileDialog::DisplayMode;
	using FileMode = FileDialog::FileMode;
```

Reply:
Ah, ok. It doesn't really matter in this case, but it would look nicer if it was easy :-)